backend/app/services/implementations/log_records_service.py

```diff
@@ -86,9 +99,9 @@ def filter_by_date_range(self, date_range):
         return f"\ndatetime>='{start_date}' AND datetime<='{end_date}'"
 
     def filter_by_tags(self, tags):
-        sql_statement = f"\n'{tags[0]}'=ANY (tags)"
+        sql_statement = f"\n'{tags[0]}'=ANY (tag_names)"
```

Comment:
when filtering, I'm noticing log records appear when they shouldn't. In this example, this is the query param: `filters={"tags":["tagA", "TagB"]}`

Here is the response:
```
{
    "log_records": [
        {
            "attn_to": 1,
            "attn_to_first_name": "Safwaan",
            "attn_to_last_name": "Chowdhury",
            "building": "Building D",
            "datetime": "2023-06-13 15:15:40.849777-04:00",
            "employee_first_name": "Safwaan",
            "employee_id": 1,
            "employee_last_name": "Chowdhury",
            "flagged": false,
            "log_id": 8,
            "note": "This is a note",
            "resident_first_name": "Saf",
            "resident_last_name": "Waan",
            "tags": [
                "tagA"
            ]
        },
        {
            "attn_to": 1,
            "attn_to_first_name": "Safwaan",
            "attn_to_last_name": "Chowdhury",
            "building": "Building D",
            "datetime": "2023-06-13 15:04:12.437682-04:00",
            "employee_first_name": "Safwaan",
            "employee_id": 1,
            "employee_last_name": "Chowdhury",
            "flagged": false,
            "log_id": 7,
            "note": "This is a note",
            "resident_first_name": "Saf",
            "resident_last_name": "Waan",
            "tags": [
                "tagA",
                "tagB"
            ]
        }
    ],
    "num_results": 2
}
```

The first log record shouldn't appear, yes it does have tag A, but it doesn't have tag B. We should only show log records that have both tag A and tag B, and other tags if any. For example, suppose another log record existed with A, B, C- that log record should appear if A and B are filtered for. 

Sorry if I explained it poorly the first time 😖